internal/mongodoc/package_test.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2015 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package mongodoc_test // import "gopkg.in/juju/charmstore.v5-unstable/internal/mongodoc"
```

Comment:
If in doubt, run github.com/rogpeppe/misc/cmd/importcomment.

I think it's reasonable to have the comment on all files, as there's no "primary" file in any given package.